darts/models/forecasting_model.py

```diff
@@ -101,55 +100,33 @@ class UnivariateForecastingModel(ForecastingModel):
     """
 
     @abstractmethod
-    def fit(self, series: TimeSeries, component_index: Optional[int] = None) -> None:
+    def fit(self, series: TimeSeries) -> None:
         """ Fits/trains the univariate model on selected univariate series.
 
         Parameters
         ----------
         series
-            The training time series on which to fit the model.
-        component_index
-            Optionally, a zero-indexed integer indicating the component to use if a multivariate
-            time series is passed.
+            A **univariate** training time series on which to fit the model.
         """
-
-        raise_if_not(series.width == 1 or (component_index is not None), "If a multivariate series is given"
-                     "as input to this univariate model, please provide a `component_index` integer indicating"
-                     " which component to use.", logger)
-
-        if series.width == 1:
-            super().fit(series)
-        else:
-            super().fit(series.univariate_component(component_index))
+        series._assert_univariate()
+        super().fit(series)
 
 
 class MultivariateForecastingModel(ForecastingModel):
     """ The base class for multivariate forecasting models.
     """
 
     @abstractmethod
-    def fit(self, series: TimeSeries, target_indices: Optional[List[int]] = None) -> None:
+    def fit(self, covariate_series: TimeSeries, target_series: TimeSeries) -> None:
         """ Fits/trains the multivariate model on the provided series with selected target components.
 
         Parameters
         ----------
-        series
-            The training time series on which to fit the model.
-        target_indices
-            A list of integers indicating which component(s) of the time series should be used
-            as targets for forecasting.
+        covariate_series
+            The training time series on which to fit the model (can be multivariate or univariate).
+        target_series
+            The target values used ad dependent variables when training the model
```

Comment:
The target can also be multivariate or univariate. I would unify the comments here.